apps/website/src/routes/docs/headless/navigation-bar/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { component$, $, useStylesScoped$ } from '@builder.io/qwik';
+import { NavigationBar } from '@qwik-ui/headless';
+
+export default component$(() => {
+  useStylesScoped$(`
+    .panel {
+      display: flex;
+      gap: 8px;
+      flex-wrap: wrap;
+    }`);
+
+  return (
+    <>
+      <h2>This is the documentation for the Navigation Bar</h2>
+
+      <div class="flex flex-col gap-8 mt-4">
```

Comment:
Headless documentation should not contain daisy classes. You have to create a custom CSS file for that. 
You can look at button-group example. Thanks